nevergrad/optimization/optimizerlib.py

```diff
@@ -1815,6 +1816,139 @@ def __init__(
 )
 
 
+class _BayesOptim(base.Optimizer):
+    def __init__(
+        self,
+        parametrization: IntOrParameter,
+        budget: tp.Optional[int] = None,
+        num_workers: int = 1,
+        *,
+        init_budget: tp.Optional[int] = None,
+        pca: tp.Optional[bool] = False,
+        n_components: tp.Optional[float] = 0.95,
+        prop_doe_factor: tp.Optional[float] = None,
+    ) -> None:
+        super().__init__(parametrization, budget=budget, num_workers=num_workers)
+
+        from bayes_optim.extension import PCABO as PcaBO
+
+        # from bayes_optim.extension import RealSpace
+        from bayes_optim import BO as BayesOptimBO
+        from bayes_optim import RealSpace
+        from bayes_optim.surrogate import GaussianProcess
+
+        lb, ub = 1e-7 - np.pi / 2, np.pi / 2 - 1e-7
+        space = RealSpace([lb, ub]) * self.dimension
+
+        # hyperparameters of the GPR model when pca = False
+        thetaL = 1e-10 * (ub - lb) * np.ones(self.dimension)
+        thetaU = 10 * (ub - lb) * np.ones(self.dimension)
+        model = GaussianProcess(thetaL=thetaL, thetaU=thetaU)  # create the GPR model
```

Comment:
if this is only useful for pca = False, then move it there

Reply:
Done